tensorflow-core/tensorflow-core-api/src/main/java/org/tensorflow/Session.java

```diff
@@ -174,11 +287,9 @@ public Runner feed(String operation, Tensor<?> t) {
      * @return this session runner
      */
     public Runner feed(String operation, int index, Tensor<?> t) {
-      Operation op = operationByName(operation);
-      if (op != null) {
-        inputs.add(op.output(index));
-        inputTensors.add(t);
-      }
+      Operation op = graph.operationOrError(operation);
```

Comment:
This is a behaviour change as before it used to silently not add anything if the operation didn't exist, now it throws an exception. This should be documented.

Reply:
It didn't actually, since `operationByName` (which is now `Graph#operationOrError`) would throw if it was going to return null.